runtime/gc_modron_standard/RootScannerReadBarrierVerifier.cpp

```diff
@@ -61,12 +61,14 @@ MM_RootScannerReadBarrierVerifier::scanConstantPoolObjectSlots(MM_EnvironmentBas
 		while (NULL != (clazz = classHeapIterator.nextClass())) {
 
 			volatile omrobjectptr_t *slotPtr = NULL;
-			GC_ConstantPoolObjectSlotIterator constantPoolObjectSlotIterator((J9JavaVM*)omrVMThread->_vm->_language_vm, clazz, true);
+#if defined(J9VM_GC_MODRON_SCAVENGER)
+			GC_ConstantPoolObjectSlotIterator constantPoolObjectSlotIterator((J9JavaVM*)omrVMThread->_vm->_language_vm, clazz);
 			while (NULL != (slotPtr = (omrobjectptr_t*)constantPoolObjectSlotIterator.nextSlot())) {
 				if (NULL != *slotPtr) {
 					doConstantPoolObjectSlots((omrobjectptr_t *)slotPtr);
 				}
 			}
+#endif /* J9VM_GC_MODRON_SCAVENGER */
```

Comment:
I believe `GC_ClassIterator` should be used here as well

Reply:
update the code to replace poisonConstantPoolObjects(), poisonStaticClassSlots()
	 with poisonClass() for ReadBarrierVerifier